src/OrchardCore.Commerce.Web/appsettings.Development.json

```diff
@@ -1,9 +1,8 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
```

Comment:
[.NET 6 change again.](https://docs.microsoft.com/en-us/aspnet/core/migration/50-to-60?view=aspnetcore-6.0&tabs=visual-studio#:~:text=log%20level%20defaults%20have%20changed%20in%20appsettings.json%20and%20appsettings.development.json%3A)